build.sbt

```diff
@@ -128,6 +128,7 @@ lazy val transformerKinesis = project
   .in(file("modules/transformer-kinesis"))
   .settings(BuildSettings.transformerKinesisBuildSettings)
   .settings(addCompilerPlugin(Dependencies.betterMonadicFor))
+  .settings(excludeDependencies ++= Dependencies.commonStreamTransformerExclusions)
```

Comment:
I don't know why this line is needed.  But without this line the dependencies are not excluded from the final asset.